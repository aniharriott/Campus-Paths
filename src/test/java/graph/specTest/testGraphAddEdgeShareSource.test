# Test Graph addEdge share source file.
# Tests if two edges can be added that
# have the same source node but different
# destination nodes.

# Create a graph
CreateGraph graph1

# Add nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

# Add edges
AddEdge graph1 n1 n2 e1
AddEdge graph1 n1 n3 e2