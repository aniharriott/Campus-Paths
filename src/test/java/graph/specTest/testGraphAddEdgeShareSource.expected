# Test Graph addEdge share source file.
# Tests if two edges can be added that
# have the same source node but different
# destination nodes.

# Create a graph
created graph graph1

# Add nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

# Add edges
added edge e1 from n1 to n2 in graph1
added edge e1 from n1 to n3 in graph1